src/components/Footer/ListLinks.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Link from './Link';
+
+const ListLinks = ({classNameContainer, classNameChild, children, links}) => (
+  <div className={classNameContainer}>
+    {children}
+    {links.map(link => (
+      <Link key={link.text || link.image} className={classNameChild}>
+        {link.image && <img className={`${classNameChild}--img`} src={link.image} alt={link.text}/>}
```

Comment:
`classNameChild`?

Isn't it too flexible? Maybe ListLink should have its styles and be reusable with the same manners and view everywhere?

Reply:
fixed